### PR TITLE
November Release Notes

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

November Release Notes.

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

**Comment by lbadenhop** [comment]
@cbordier Ready for content.

File: content/en/docs/amplify_relnotes/20211130_ampc_relnotes.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+The following enhancements are available in this Axway CLI update:
+
+* Axway CLI has added the current team/role to the 'auth list', 'auth login', and 'auth whoami' commands.
+* Axway CLI has added an 'axway auth swtich --team name' command to switch your current team to another team which you are a member of.
+
+The following enhancements are available in this Axway Central CLI update:
+
+* Amplify Central CLI can manage team ownership of Environments and API Services.
+* Amplify Central CLI has added a simple '--team' filter for users to only Get Environments and API Services for a specific team which they are a member of.
```
Comment (jquick-axway) [inline content/en/docs/amplify_relnotes/20211130_ampc_relnotes.md:31]: The `--team` feature did **not** make it into the Central CLI 2.5.3 release. It'll be in the next release.

**Review by cbordier — APPROVED** [review]
I approve the changes
@lbadenhop, you can merge

File: content/en/docs/amplify_relnotes/20211130_ampc_agents_relnotes.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+* None
+
+### Amplify Azure agents enhancements
+
+* None
+
+### Amplify Istio agents enhancements
+
+* Add text
```
Comment (robertengaxway) [inline content/en/docs/amplify_relnotes/20211130_ampc_agents_relnotes.md:42]: Set this to 'None'

**Review by robertengaxway — APPROVED** [review]
I approve

**Pull request merged**